specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/compute.json

```diff
@@ -6888,6 +7381,300 @@
     }
   },
   "definitions": {
+    "RestorePointCollectionSourceProperties": {
+      "properties": {
+        "location": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Location of the source resource used to create this restore point collection."
+        },
+        "id": {
+          "type": "string",
+          "readOnly": true,
```

Comment:
id in RestorePointCollectionSourceProperties is also needed as an input when creating the RestorePointCollection, hence it can't be set to readOnly = true.